docs/landing-zone-v2/README.md

```diff
@@ -615,6 +616,70 @@ nomos status --contexts gke_${PROJECT_ID}_northamerica-northeast1_krmapihost-${C
       --role "roles/billing.user"
     ```
 
+### <a name='Optional-ConfigControllerMonitoring'></a>Optional - Config Controller Monitoring
+
+Proceed with this section ONLY IF monitoring of Policy Controller and/or Config Sync metrics on the Config Controller cluster is required.  Refer to official documentation for more details.
+
+> **WARNING: once enabled, billable metrics will be generated.**
+
+Follow these steps after the `core-landing-zone` package is applied which deploys most resources required (GCP SA, IAM, etc.).
+
+Alert policies, dashboards, etc. based on these metrics will need to be created manually.
+
+- To enable [Config Sync metrics](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync) with Cloud Monitoring, annotate the kubernetes `default` service account in the `config-management-monitoring` namespace with the GCP `config-mgmt-mon-default-sa` service account:
+    ```bash
+    export PROJECT_ID='<management-project-id>'
+
+    # annotate the service account to enable monitoring
+    kubectl annotate serviceaccount \
+      --namespace config-management-monitoring \
+      default \
+      iam.gke.io/gcp-service-account=config-mgmt-mon-default-sa@${PROJECT_ID}.iam.gserviceaccount.com
+
+    # verify output for proper annotation
+    kubectl describe serviceaccount default --namespace config-management-monitoring
+
+    # if metrics are not being generated, the 'otel-collector' pod may need to be restarted:
+    kubectl rollout restart deployment otel-collector -n config-management-monitoring
+    ```
+    Note from [official documentation](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync#metric_labels):
+    > **Note:** Since Config Sync version 1.15.0, depending on the frequency of production repository commits and variance in Kubernetes Kinds being deployed, the two new metric labels `commit` and `type` introduced in this release can cause an increase in Cloud Monitoring metric write throughput. This can result in an increase of monthly billing. The affected metrics are `apply_duration_seconds`, `api_duration_seconds`, `declared_resources`, `apply_operations_total`. To avoid an increase in billing or if these metrics are not needed, follow the instructions to [patch the otel collector deployment with ConfigMap](https://github.com/GoogleContainerTools/kpt-config-sync/blob/v1.15/docs/custom-metric-filter.md#patch-the-otel-collector-deployment-with-configmap) and remove the aforementioned metrics from `filter/cloudmonitoring` allowlist.
+
+    If monitoring is no longer required, the annotation can be removed with the command below:
+    ```bash
+    # remove the annotation
+    kubectl annotate serviceaccount \
+      --namespace config-management-monitoring \
+      default \
+      iam.gke.io/gcp-service-account-
```

Comment:
Looking up the kubectl call but I thnk the service account is truncated before and expected gcp-service-account=...
at gcp-service-account-

Reply:
never mind - I checked, to remove the SA place a - minus at the end
all good - adding this one to my queue

https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity